bad-flask-app/requirements.txt

```diff
@@ -1,8 +1,9 @@
 Click==7.0
-Flask==1.0.2
+Flask==1.1.1
```

Comment:
Why 1.1.1 was chosen and not 1.1.4 which is latest 1.1.x? Latest in pypi is 2.1.1 but I understand that we don't want to update to major version

Reply:
Answer is in the description:
> In this pull request, libraries are updated to only as new as necessary to fix the issue (to reduce the risk of introducing more breaking changes).

I tried at least 3 different versions of Flask (1.0.4, 1.1.0, 1.1.1), and found 2 were not working. The version of Flask needs to be synced with Werkzeug (apparently), or the latter breaks. I found that 1.1.1 worked with the newest Werkzeug fixed for collections import.
I also had limited time to find the combo of versions that worked, so I went with these, rather than check all the possible options.

Reply:
Actually, it's second newest Werkzeug version, as the one that had collections fixed would not work with any of the 3 Flasks.